mobile/cmp/navigator/NavigatorPageModel.js

```diff
@@ -34,26 +40,55 @@ export class NavigatorPageModel {
         return stringify({id, props}, qsOpts);
     }
 
+    get renderMode() {
+        return this._renderMode || this.navigatorModel.renderMode;
```

Comment:
we should prefer nullish coalesce ('??') here (and I realize this code is copied from some similar objects)